documentation/API-onboarding.md

```diff
@@ -0,0 +1,75 @@
+
+![API process pipeline](ressources/API_overview_list.png)
+
+Details and clarifications on the stages conforming this pipeline are provided in the following sections.
+
+## Submission of an API proposal
+
+The pipeline is triggered with the submission of an API proposal. This API proposal can represent a new API or a new API family.
+
+API proposals can be submitted by everybody, e.g. GSMA OPAG, external or CAMARA member companies, and existing CAMARA Sub Projects or Working Groups (if these realize that the scope has to be increased).
+
+To proceed with the submission, the API proposal owner shall follow these steps:
+
+- Fill in the template available [here](https://github.com/camaraproject/WorkingGroups/blob/main/Commonalities/documentation/API-proposal-template.md) and save it with the following name: "APIproposal\_\<APIname\>\_\<owner\>. md" locally.
+- Create a new issue in the API Backlog Working Group repository, labeled with "API Backlog".
+- Upload the filled-in template to [GitHub repository folder for API proposals](https://github.com/camaraproject/WorkingGroups/tree/main/APIBacklog/documentation/SupportingDocuments/API%20proposals) via pull request. This pull request shall be associated to the issue created in the previous step.
+
+## Evaluation of the API proposal
+
+Upon submission, the "API Backlog" Working Group (WG) validates the API proposal. In particular:
+
+- The API Backlog WG coordinator checks that the template is duly filled in. Otherwise, the API proposal owner will be requested to provide missing information.
+- After this sanity check, each WG participant declares his/her support (which is also documented in the template).
+
+Finally the pull request is merged into the main branch and the API proposal template is sent to the Steering Committee of CAMARA at least one week prior to the Steering Committee meeting where the API proposal shall be voted upon.
+
+This whole step shall be done within 2 regular meetings of the API Backlog WG. In parallel, the API proposal is formally registered in CAMARA by adding it to the [CAMARA API overview list](https://github.com/camaraproject/WorkingGroups/blob/main/APIBacklog/documentation/APIBacklog.md) (API Backlog WG creates a new line in the table and fills in API family name, API family owner, API family proposal registration date, the link to the application template and the supporters for it).
+
+## API proposal Steering Committee decision
+
+Upon receiving the API proposal and notification from the API Backlog WG, the Steering Committee studies the proposal and votes it at the Steering Committee meeting.
+
+- No-Go! Objecting companies shall provide justifications why. The API proposal is rejected and will not be included in any API Sub Project.
+- Go! In this case, the Steering Committee shall specify whether the API proposal is to be hosted by a new or existing Sub Project.
+
```

Comment:
Should there be any reference to the voting rules for th enumber of voters e.g. I assume one No Go vote would not block an API submission. So I thing one of the voting rules should be referenced / applied for the API voting.

Reply:
I think we need to have this reference but at the same time we also should start to work towards creating a TSC and  new API proposals should be decided on by TSC

Reply:
@tanjadegroot : The voting is already descriped in our Project charter. That also applies for this vote, we shouln't document it twice and redundant.
@nathanrader : correct. Will be included in our new governance proposal.

Reply:
No intent to duplicate, just making sure the right process for API suppport decisions is available and complete :-)